portal-kernel/src/com/liferay/exportimport/kernel/lar/StagedModelDataHandler.java

```diff
@@ -73,23 +158,91 @@ public void importCompanyStagedModel(
 			PortletDataContext portletDataContext, String uuid, long classPK)
 		throws PortletDataException;
 
+	/**
+	 * "Imports" missing reference. When a reference is exported as missing, the
+	 * framework calls this method during the import process. This method
+	 * updates the new primary key map in <code>portletDataContext</code>. In
+	 * other words it maps the ID of the existing staged model to the old ID
+	 * which is in the <code>referenceElement</code>.
+	 *
+	 * @param  portletDataContext the portlet data context of the current
+	 *         process
+	 * @param  referenceElement XML element that contains information about the
+	 *         reference
+	 * @throws PortletDataException
+	 */
 	public void importMissingReference(
 			PortletDataContext portletDataContext, Element referenceElement)
 		throws PortletDataException;
 
+	/**
+	 * When this <code>StagedModelDataHandler</code> extends {@link
+	 * BaseStagedModelDataHandler}, {@link
+	 * BaseStagedModelDataHandler#importMissingReference(PortletDataContext,
+	 * Element)} and {@link
+	 * BaseStagedModelDataHandler#doImportMissingReference(PortletDataContext,
+	 * Element)} are not overridden, this method is called.
```

Comment:
@matethurzo I don't understand what we're trying to say in this description. I modified it to say:

> Imports the missing staged model reference. When the staged model data handler instance extends `BaseStagedModelDataHandler`, this method is called to override the `BaseStagedModelDataHandler#importMissingReference(PortletDataContext, Element)` and `BaseStagedModelDataHandler#doImportMissingReference(PortletDataContext, Element)` methods.

Is that correct?